aiqleads/core/project_tracking.py

```
@@ -28,17 +34,75 @@
     def _load_project_structure(self) -> Dict:
         """Load project structure from PROJECT_STRUCTURE.md."""
         try:
-            structure_path = os.path.join("docs", "PROJECT_STRUCTURE.md")
-            with open(structure_path, 'r') as f:
-                content = f.read()
-            # Parse the markdown structure (simplified for now)
-            return {"root": "aiqleads"}
+            root = Path(__file__).parent.parent.parent
+            structure_path = os.path.join(root, "docs", "PROJECT_STRUCTURE.md")
+            if os.path.exists(structure_path):
+                with open(structure_path, 'r') as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)